docs/.gitignore

```diff
@@ -9,4 +9,8 @@ _site/
 vendor/
 
 # Swagger
-estela/api/endpoints.html
\ No newline at end of file
+estela/api/endpoints.html
+
+# Ruby
```

Comment:
Sure to add this to gitignore patterns?

Reply:
I removed this to maintain it same as main branch.